query.go

```diff
@@ -574,3 +577,140 @@ func aurInfoPrint(names []string) ([]*rpc.Pkg, error) {
 
 	return info, nil
 }
+
+func aurPkgbuilds(names []string) ([]string, error) {
+	pkgbuilds := make([]string, 0, len(names))
+	var mux sync.Mutex
+	var wg sync.WaitGroup
+	var errs multierror.MultiError
+
+	makeRequest := func(name string) {
+		defer wg.Done()
+
+		values := url.Values{}
+		values.Set("h", name)
+
+		url := "https://aur.archlinux.org/cgit/aur.git/plain/PKGBUILD?"
+
+		resp, err := http.Get(url + values.Encode())
+		if err != nil {
+			errs.Add(err)
+			return
+		}
+		defer resp.Body.Close()
+
+		body, readErr := ioutil.ReadAll(resp.Body)
+		pkgbuild := string(body)
+
+		if readErr != nil {
+			errs.Add(readErr)
+			return
+		}
+
+		if strings.Contains(pkgbuild,
+			"<div class='content'><div class='error'>Invalid branch: "+name+"</div>") {
+			errs.Add(fmt.Errorf("package \"%s\" was not found", name))
+			return
+		}
+
+		mux.Lock()
+		pkgbuilds = append(pkgbuilds, pkgbuild)
+		mux.Unlock()
+	}
+
+	for _, name := range names {
```

Comment:
Not blocking: I would prefer it to be done in chunks of 20? let's say because if someone adds 100 targets it will be a bit of a mess, but it's a lot of work for a niche edge case.

Reply:
What do you mean by "mess"? The PKGBUILDs are requested concurrently but written in sync.

Reply:
Sometimes when we open too many concurrent connections against the AUR we get rate limit banned, which is why yay chunks request making, just warning that this seems the same unlimited pattern that gave us that problem,